en/docs/install-and-setup/setup/kubernetes-deployment/kubernetes/am-pattern-4-acp-tm-gw-km.md

```
@@ -32,10 +32,10 @@ This is the fully distributed deployment for API Manager. The default configurat
     - [3. Traffic Manager Configurations](#3-traffic-manager-configurations)
           - [3.1 Configure Key Manager and Eventhub](#31-configure-key-manager-and-eventhub)
           - [3.2 Deploy Traffic Manager](#32-deploy-traffic-manager)
-    - [4. Universal Gateway Configuration](#4-universal-gateway-configuration)
+    - [4. Classic Gateway Configuration](#4-universal-gateway-configuration)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)